go.sh

```diff
@@ -235,3 +235,6 @@ if [ "${STATSD_METRICS_ENABLED}" = "TRUE" ]; then
 fi
 
 eval "${NGINX_BIN} -g \"daemon off;\""
+# Use SIGQUIT instead of default SIGTERM to cleanly drain requests
+# See https://github.com/openresty/docker-openresty/blob/master/README.md#tips--pitfalls
+STOPSIGNAL SIGQUIT
```

Comment:
I don’t understand why you have moved the `STOPSIGNAL SIGQUIT` from the Dockerfile to this go.sh file.

Is it possible to give a short explanation? 
Was it not working the other way?
Is this better in some way?

Reply:
without STOPSIGNAL SIGQUIT, it worked on carrier portal. 
This recommendation has come from public repo. I really do not want to add it in the first instance. I read through the justification and it actually made sense, so I added it back.
 reference https://github.com/openresty/docker-openresty/blob/master/README.md#tips--pitfalls